cdec/installation-manager-core/src/main/java/com/codenvy/im/facade/IMArtifactLabeledFacade.java

```diff
@@ -120,8 +120,8 @@ public IMArtifactLabeledFacade(@Named("installation-manager.download_dir") Strin
 
     /** {@inheritDoc} */
     @Override
-    public List<UpdateArtifactInfo> getAllUpdates(@Nullable Artifact artifact) throws IOException, JsonParseException {
-        List<UpdateArtifactInfo> updates = super.getAllUpdates(artifact);
+    public List<UpdateArtifactInfo> getAllUpdates(@Nullable Artifact artifact, boolean fromInstalledVersion) throws IOException, JsonParseException {
```

Comment:
I would like to have 2 methods:
* getAllUpdates()
* getUpdatesAfterVersion()
It is more readable.